Source/resources/mongoDB/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+export { MongoDBConnectionString } from './MongoDBConnectionString';
+export { IMongoDBResource } from './IMongoDBResource';
+export * as internal from './internal';
```

Comment:
Do we usually export the internals like this? They aren't really ment to be used externally...

Reply:
Seems like we need it for now - so let's just leave it.